<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDCSource</class>
 <widget class="QMainWindow" name="FrmDCSource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>115</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DC Source Settings</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel4_2">
        <property name="text">
         <string>Channel</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbChannel"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="spacer2_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="spacer5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="spacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Value</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_edValue"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Function</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbFunction"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel4_3">
        <property name="text">
         <string>Range</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cmbRange"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="m_ckbOutput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="spacer3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_cmbChannel</tabstop>
  <tabstop>m_ckbOutput</tabstop>
  <tabstop>m_cmbFunction</tabstop>
  <tabstop>m_edValue</tabstop>
  <tabstop>m_cmbRange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
